Makefile

```diff
@@ -129,4 +137,21 @@ docs:
 	rm -rf docs/_build/
 	@${ACTIVATE} sphinx-build docs/ docs/_build/
 
+keycloak:
+	@echo "-> Run Keycloak Identity provider"
+	docker run -dit --name scancodeio-keycloak \
+                    --env KEYCLOAK_USER=${KEYCLOAK_USER} \
+                    --env KEYCLOAK_PASSWORD=${KEYCLOAK_PASSWORD} \
+                    --publish ${KEYCLOAK_HTTPS_BIND_PORT}:8443 \
+                    jboss/keycloak
+	@echo "-> Keycloak launched. Please visit https://localhost:${KEYCLOAK_HTTPS_BIND_PORT} and configure realm"
```

Comment:
add script to configure using kcadm.sh
https://github.com/keycloak/keycloak-documentation/blob/master/server_admin/topics/admin-cli.adoc